core/pom.xml

```diff
@@ -56,7 +56,7 @@
     <repositories>
         <repository>
             <id>confluent</id>
-            <url>http://packages.confluent.io/maven/</url>
+            <url>https://packages.confluent.io/maven/</url>
```

Comment:
Thanks for noticing. Appreciate the fix.
However, this particular connection to packages.confluent.io/maven is redundant and never used for resolving. It's better to comment this out with HTTPS(not delete it - because this prompts Kafka users to use this connection when they use Schema Registry of Confluent packages)